lib/pedant_mysql2.rb

```diff
@@ -46,7 +48,15 @@ def whitelist
     end
 
     def ignored?(warning)
-      whitelist.any? { |matcher| warning.message.match?(matcher) } || drop_table_warning(warning)
+      return true if whitelist.any? { |matcher| warning.message.match?(matcher) }
+      return true if note_warning?(warning)
+      return true if drop_table_warning(warning)
```

Comment:
I believe both `(1051) Unknown table 'foo'` and `(1050)  Table 'foo' already exists` are note-level warnings, so we should be able to remove the `drop_table` specific check.

Reply:
Fixed. Left both tests in to ensure the behavior didn't change.